tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Graph.java

```diff
@@ -545,18 +545,21 @@ private synchronized void addInitOp() {
     if (!newInitializers) {
       return;
     }
-    if (initializers.isEmpty()) return;
+    if (initializers.isEmpty()) {
+      return;
+    }
 
     OperationBuilder builder = baseScope().opBuilder(NoOp.OP_NAME, INIT_OP_BASE_NAME);
     initializers.forEach(builder::addControlInput);
     builder.build();
-
     newInitializers = false;
   }
 
   /**
    * Generate a representation of the Graph.
    *
+   * <p>Will create an initialization operation that depends on all registered init ops.
```

Comment:
Maybe "If there are newly registered initializers on this graph this call adds an initialization operation to this graph that depends...". The current text isn't explicit that it mutates the graph, nor does it state when it does that.